jenkins/run-server-func-tests

```diff
@@ -29,7 +70,7 @@ function dump_journal {
 }
 
 function cleanup {
-    if [[ -n "${cleanup_flag}" ]]; then
+    if (( "${cleanup_flag}" )); then
```

Comment:
Are you sure that you want to convert `cleanup_flag` to a string inside a numerical context?  Ditto line 144.

Reply:
Left too much behind. It actually doesn't matter, but it's not necessary.